data/categories.txt

```diff
@@ -7032,7 +7035,7 @@ ar:^ﺓﺭﺎﻴﺴﻟﺍ ءﺍﺰﺟﺃ
 cs:^Autodíly
 da:^Bildele
 nl:^Auto onderdelen
-fi:^Auton osat
+fi:^Autotarvikeet
```

Comment:
Autotarvikkeet?

Reply:
Well maybe "Auton osat" were better translation than new one. Reverted that.